### PR TITLE
Unit test dpp dps management

### PR DESCRIPTION
**Pull request by EmmaRenauld**
# Quick description

More units tests for the tractogram module.

Note: The correlation option only worked in the endpoint method. Removed from the OPERATIONS dict.

## Type of change

Check the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Provide data, screenshots, command line to test (if relevant)

...

# Checklist

- [ ] My code follows the style guidelines of this project (run [autopep8](https://pypi.org/project/autopep8/))
- [ ] I added relevant citations to scripts, modules and functions docstrings and descriptions
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I moved all functions from the script file (except the argparser and main) to scilpy modules
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by grahamlittlephd — APPROVED** [review]
This pull request all looks good to me other than two minor typos. Tested the newly added tests as well as some ran some "home cooked" tests from my original pull.

Nice work! Also thanks for doing all the hard work filling in my blank unit tests, much appreciated @EmmaRenauld you are a scilpy legend!

File: scilpy/tractograms/dps_and_dpp_management.py [hunk 1]
```diff
@@ -301,58 +320,49 @@ def perform_operation_dpp_to_dps(op_name, sft, dpp_name, endpoints_only=False):
...
     sft: StatefulTractogram
         The streamlines used in the operation.
     dpp_name: str
         The name of the data per point to be used in the operation.
 
     Returns
     -------
-    new_sft: StatefulTractogram
-        sft with data per streamline resulting from the operation.
+    new_dpp: List
```
Comment (arnaudbore) [inline scilpy/tractograms/dps_and_dpp_management.py:336]: put new_ddp at L340/344 or new_data_per_streamline here

File: scilpy/tractograms/dps_and_dpp_management.py [hunk 2]
```diff
@@ -126,6 +140,7 @@ def project_map_to_streamlines(sft, map_volume, endpoints_only=False):
     else:
         dimension = 1
 
+    # toDo: double loop. Could be faster?
```
Comment (arnaudbore) [inline scilpy/tractograms/dps_and_dpp_management.py:143]: why would you add a second loop here ? I don't undersand :/

**Review by arnaudbore — APPROVED** [review]
Quick comments but GTG

File: scilpy/tractograms/dps_and_dpp_management.py [hunk 3]
```diff
@@ -4,25 +4,29 @@
...
+def add_data_as_color_dpp(sft, cmap, data, clip_outliers=False, min_range=None,
+                          max_range=None, min_cmap=None, max_cmap=None,
+                          log=False, LUT=None):
     """
     Normalizes data between 0 and 1 for an easier management with colormaps.
     The real lower bound and upperbound are returned.
 
     Data can be clipped to (min_range, max_range) before normalization.
-    Alternatively, data can be kept as is,
+    Alternatively, data can be kept as is, but the colormap be fixed to
```
[the remark this answers is not in the document]
Comment (EmmaRenauld) [reply scilpy/tractograms/dps_and_dpp_management.py:15]: Hmm, that's not exactly it. I would you say nicely something like "Alternatively, the data can be kept as is, but instead user can fix the colormap between (min_cmap, max_cmap)"

**Review by arnaudbore — APPROVED** [review]

**Pull request merged**